src/main/java/List.java

```diff
@@ -0,0 +1,31 @@
+public class List {
+    //array of 100 size
+    private String[] tasks = new String[100];
+
+    public static int totalTasks = 0;
+
+    public List() {
+        this.tasks = new String[100];
+    }
+
+    public void getAllTasks() {
+        int serialNumber = 1;
+        for (int i = 0; i < totalTasks; i++) {
+            System.out.print("     ");
+            System.out.println(serialNumber + ". " + tasks[i]);
```

Comment:
Can consider adding the spaces together with the println for better readability as you have earlier